src/javascript/_common/utility.js

```diff
@@ -40,6 +40,41 @@ const getHighestZIndex = (selector = 'div,p,area,nav,section,header,canvas,aside
     return all.length ? Math.max(...all) : null;
 };
 
+const eu_countries = [
+    'it',
+    'de',
+    'fr',
+    'lu',
+    'gr',
+    'mf',
+    'es',
+    'sk',
+    'lt',
+    'nl',
+    'at',
+    'bg',
+    'si',
+    'cy',
+    'be',
+    'ro',
+    'hr',
+    'pt',
+    'pl',
+    'lv',
+    'ee',
+    'cz',
+    'fi',
+    'hu',
+    'dk',
+    'se',
+    'ie',
+    'im',
+    'gb',
+    'mt',
+];
+// check if client is from EU
+const isEuCountryNoIp = country => eu_countries.includes(country);
```

Comment:
I have a suggestion for the function name, please let me know if you agree, otherwise the changes are fine.